pkgs/tools/misc/systrayhelper/default.nix

```diff
@@ -29,6 +35,20 @@ buildGoPackage rec {
   nativeBuildInputs = [ pkgconfig libappindicator-gtk3 ];
   buildInputs = [ libappindicator-gtk3 ];
 
+  enableParallelBuilding = true;
+
+  checkInputs = [ xvfb_run dbus i3 xdotool ];
+  doCheck = stdenv.isLinux;
+  # use the override.nix from the source to extract and watch tests
+  checkPhase = ''
+    export PATH=$PATH:$(pwd)/go/bin:${i3}/bin:${xdotool}/bin
```

Comment:
Are `i3` and `xdotool` necessary here? I think `checkInputs` should add them to `$PATH` automatically.